tensorflow/python/ops/variable_scope.py

```diff
@@ -887,6 +887,14 @@ def get_collection(self, name):
     scope = self._name + '/' if self._name else self._name
```

Comment:
In the else-branch, is self._name an empty string? None? Could we just use a constant after else here?